handler.go

```diff
@@ -46,6 +46,10 @@ type Spec struct {
 	// Vulcan middlewares to register with the handler. When registering, middlewares are assigned priorities
 	// according to their positions in the list: a middleware that appears in the list earlier is executed first.
 	Middlewares []middleware.Middleware
+
+	// When Handler or HandlerWith body is used, this function will be called after every request with a log message.
+	// If nil, defaults to github.com/mailgun/log.Infof.
+	Logger func(format string, a ...interface{})
```

Comment:
Actually, how about instead of this we make this configuration parameter more specific, like:

```go
LogEveryRequest bool
```

I just can hardly imagine this log.Info override be useful in any other case.